src/cdk/a11y/key-manager/list-key-manager.ts

```diff
@@ -265,7 +265,7 @@ export class ListKeyManager<T extends ListKeyManagerOption> {
   }
 
   /** Index of the currently active item. */
-  get activeItemIndex(): number | null {
+  get activeItemIndex(): number {
```

Comment:
You can avoid the breaking change here by reverting the change. If it's something that we should clean up, you can add `@breaking-change 10.0.0 remove `null` from return type` and we'll do it next time we're making breaking changes.